tools/bmh_test/run_local_bmh_test_setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -eux
+
+REPO_ROOT=$(realpath "$(dirname "${BASH_SOURCE[0]}")/../..")
+
+cd "${REPO_ROOT}" || exit 1
+
+# List of packages to check
+packages=("virtinst" "libvirt-clients" "libvirt-daemon-system")
+
+# Check each package
+for pkg in "${packages[@]}"; do
+    if ! dpkg -s "$pkg" >/dev/null 2>&1; then
+        echo "ERROR: $pkg is not installed. Please install the required package."
+        exit 1
+    fi
+done
```

Comment:
I think it would be enough to check if `virsh` and `virt-install` is available. Then the script will also work with mac and other linux distros.

```suggestion
# List of packages to check
commands=("virt-install" "virsh")

# Check each package
for cmd in "${commands[@]}"; do
    if ! command -v "${cmd}" &> /dev/null; then
        echo "ERROR: ${cmd} not found. Please install it."
        exit 1
    fi
done
```